cdap-api-common/src/main/java/io/cdap/cdap/api/data/schema/Schema.java

```diff
@@ -50,6 +51,9 @@ public final class Schema implements Serializable {
   private static final long serialVersionUID = -1891891892562027345L;
   private int precision;
   private int scale;
+  private ByteBuffer fixedValue;
```

Comment:
Why is this in schema? Schema should have no values

Reply:
+1

Reply:
Removed